config.go

```diff
@@ -67,4 +68,7 @@ func (c *Config) fillDefaults() {
 	if c.Eureka.PollIntervalSeconds == 0 {
 		c.Eureka.PollIntervalSeconds = 30
 	}
+	if c.Eureka.ServerContext == "" {
```

Comment:
- [ ] Use `len` instead.  
```go
if len(c.Eureka.ServerURLBase) == 0 {
```
- [ ] This makes it impossible to use an empty URL base.  
  Is it possible to host Eureka at the root URL?

Reply:
It's not possible. As I got from Eureka code, URL routing is based on API version at least. So, we can safely do this check.